### DISCUSSION TITLE
Contribution rules for tvdb4.mapping.xml?

### DISCUSSION THREAD

**Discussion by EarthBoundGiygas** [ZeroQI/Absolute-Series-Scanner#498] [General]
Hey I was wondering if there was any rules for contributing to the tvdb4 mapping xml file. There is not terribly many shows on there as there are out on anidb, and I know that TVDB4 is the only way to properly do a split-cour format on Plex with the arr stack.  

I found that this was the only way to do a anidb2 style mapping with the tvdb id formatting. Or should I just maintain my own personal xml file for this? Thanks!

**Comment by ZeroQI** [comment]
tvdb4 is for using sagas as seasons for super long series, for which users opt in tvdb4 mode. Users can use their own season folder or local xml format for personal files is very similar to the public xml

If you do pull requests for not present series i will approve

**Reply by EarthBoundGiygas** [reply]
Thanks, I found a few that needed to be fixed. Where should the tvdb4.mapping.xml be hosted locally in relation to the scanner to test this without relying on the github? System is using a cached file in my tempfs.

**Comment by ZeroQI** [comment]
https://github.com/ZeroQI/Absolute-Series-Scanner/blob/048e8001a525ba1c04afda2aa2005feb74709eb8/Scanners/Series/Absolute%20Series%20Scanner.py#L840

Tvdb4.mapping locally in series folder with the mapping inside, no xml tags
